src/test/java/org/opentripplanner/middleware/testUtils/CommonTestUtils.java

```diff
@@ -0,0 +1,31 @@
+package org.opentripplanner.middleware.testUtils;
+
+import org.opentripplanner.middleware.utils.FileUtils;
+
+import java.io.IOException;
+
+public class CommonTestUtils {
+    /**
+     * Whether the end-to-end environment variable is enabled.
+     */
+    public static final boolean isEndToEnd = getBooleanEnvVar("RUN_E2E");
+
+    private static final String TEST_RESOURCE_PATH = "src/test/resources/org/opentripplanner/middleware/";
+
+    /**
+     * Returns true only if an environment variable exists and is set to "true".
+     */
+    public static boolean getBooleanEnvVar(String var) {
+        String variable = System.getenv(var);
+        return variable != null && variable.equals("true");
+    }
+
+    public static <T> T getResourceFileContentsAsJSON(String resourcePathName, Class<T> clazz) throws IOException {
+        return FileUtils.getFileContentsAsJSON(TEST_RESOURCE_PATH + resourcePathName, clazz);
+    }
+
+    public static String getResourceFileContentsAsString(String resourcePathName) throws IOException {
```

Comment:
I think a better name for this method would be `CommonTestUtils#getTestResourceAsString`. Same goes for the above (`CommonTestUtils#getTestResourceAsJSON`)

Reply:
Changed in 695f54c4ffb08e23144a496d611b999c93c86764.